src/modules/fundingManager/bondingCurveFundingManager/IBancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -21,6 +21,26 @@ interface IBancorVirtualSupplyBondingCurveFundingManager {
         uint32 indexed newSellReserveRatio, uint32 indexed oldSellReserveRatio
     );
 
+    //--------------------------------------------------------------------------
+    // Structs
+    struct IssuanceToken {
+        bytes32 name;
+        bytes32 symbol;
+        uint8 decimals;
+        uint initialTokenSupply;
```

Comment:
Initial Token Supply and Initial Collateral Supply are actually Bonding Curve Parameters, since it's what is used virtually for the internal math. They'd need moving there.

Reply:
Implemented this suggestion. Can you review this again?